code/game/objects/items/pillow.dm

```diff
@@ -125,15 +125,13 @@
 	var/datum/component/two_handed/two_handed = GetComponent(/datum/component/two_handed)
 	if(two_handed)
 		AddComponent(/datum/component/two_handed, force_unwielded = two_handed.force_unwielded + 5, force_wielded = two_handed.force_wielded + 10)
-		force += 5
-	else
-		force += 5
+	force += 5
 	update_appearance()
 
 /// Smothers the victim while the do_after succeeds and the victim is laying down or being strangled
 /obj/item/pillow/proc/smothering(mob/living/carbon/user, mob/living/carbon/victim)
 	while(victim)
-		if(victim.body_position == FALSE && user.grab_state >= GRAB_AGGRESSIVE)
+		if((victim.body_position != LYING_DOWN) && ((user.grab_state < GRAB_AGGRESSIVE) || (user.pulling != victim)))
```

Comment:
Oh thanks goodness you noticed the mistake I did regarding the logic in my previous comment when I suggested the operator `<=` and not `<`.